### DISCUSSION TITLE
Firefly Does not Delete Profile Data When it is Uninstalled

### DISCUSSION THREAD

**Discussion by MauritzBotma** [iotaledger/firefly#2027] [Features / Enhancements]
## Issue description

I created a profile, uninstalled Firefly, Reinstalled Firefly and the profile created before the uninstall is still visible and I can use the pin and password to open the wallet. 

## Wallet version

1.0.0

## System specification

Windows 19042.928

## Is it a repeatable issue?

Y

## Steps To reproduce the issue

Explain how the maintainer can reproduce the issue.

1. Create a profile
2. Uninstall Firefly
3. Reinstall Firefly
4. Open Firefly

## Expected behaviour

All profile data, pins, and passwords should be deleted when Firefly is uninstalled

## Actual behaviour

The profile data is not deleted

## Errors

None

**Comment by Tsangares** [comment]
I'm not a developer. But I think this is a feature not a bug to delete the seeds upon uninstalling. You would want to manually delete them in the app first. What about accidentally uninstalling the app and realizing your tokens are not backed up? Better safe than sorry in that case. Also it depends on how you install the application. Using pacman on arch linux, you can program the package to delete the files upon uninstall. 

Opinion: This is a good feature not a bug.

**Comment by cdnthought** [comment]
Although this feature may be useful, it is clearly a bug if the option to remove all data is not given in the uninstaller.  Windows now hides this data and I for one can't find it and I need to delete the profiles as I was just helping friends migrate - I don't want their data!  Keep in mind that you can't delete profiles unless there is a 0 balance.  So this is definitely a bug, not a feature.
